logevents/pom.xml

```diff
@@ -42,6 +42,12 @@
             <version>4.13.2</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock</artifactId>
+            <version>2.27.2</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I'd prefer not to add new dependencies, even for testing, unless the value is extremely high

Reply:
wiremock is such a easy way to handle the stubbing of APIs however, but I'll see if I can replicate what has been done in https://github.com/jhannes/logevents/blob/main/logevents/src/test/java/org/logevents/observers/SlackLogEventObserverTest.java#L53 instead.

Reply:
Thanks. I think I will need to find a proper abstraction for these, but for now, this looks good in the pull request.